examples/ingress_create.py

```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""
+Creates deployment, service, and ingress objects. The ingress allows external
```

Comment:
@roycaihw see example here -

Reply:
ah I agree. I'm okay with either way as long as we are consistent.

Reply:
Updated objects to be lowercased.